portal/src/i18n/index.js

```diff
@@ -1,33 +1,51 @@
-import { startsWith, forEach } from 'lodash'
+import { startsWith } from 'lodash'
 import Vue from 'vue'
 import VueI18n from 'vue-i18n'
 import { getLocation } from '~/utils'
+import axios from 'axios'
+import injector from 'vue-inject'
 
 Vue.use(VueI18n)
 
-export async function createI18N() {
-  // Set instance options
-  const i18n = new VueI18n({ fallbackLocale: 'nl' })
-  i18n.locales = [
-    { code: 'en', iso: 'en-US', file: 'en.js' },
-    { code: 'nl', iso: 'nl-NL', file: 'nl.js' }
-  ]
-  i18n.defaultLocale = 'nl'
-  i18n.differentDomains = false
-  i18n.forwardedHost = false
-  i18n.beforeLanguageSwitch = () => null
-  i18n.onLanguageSwitched = () => null
-  i18n.locale = startsWith(getLocation(), '/en/') ? 'en' : 'nl'
-
-  // Lazy-load translations
-  const { loadLanguageAsync } = require('./utils')
+const $log = injector.get('$log')
+
+const i18n = new VueI18n({ fallbackLocale: 'nl' })
+i18n.locales = [
+  { code: 'en', iso: 'en-US', file: 'en.js' },
+  { code: 'nl', iso: 'nl-NL', file: 'nl.js' }
+]
+i18n.defaultLocale = 'nl'
+i18n.differentDomains = false
+i18n.forwardedHost = false
+i18n.beforeLanguageSwitch = () => null
+i18n.onLanguageSwitched = () => null
+i18n.loadedLanguages = []
+i18n.locale = startsWith(getLocation(), '/en/') ? 'en' : 'nl'
+
+export async function loadLanguages() {
   await loadLanguageAsync(i18n, i18n.locale)
-  forEach(i18n.locales, locale => {
-    if (locale.code === i18n.locale) {
-      return
+
+  i18n.locales
+    .filter(locale => locale !== i18n.locale)
+    .forEach(locale => loadLanguageAsync(i18n, locale.code))
+}
+
+async function loadLanguageAsync(i18n, locale) {
```

Comment:
Can we perhaps move this to the top of the file. I know JS does it, but it helps to understand the file. It can even move inside the loadLanguage function, because it's used nowhere else.

Reply:
Oops, already merged it. Yeah, sure I will change it.